packages/cli/src/commands/setup/auth/templates/auth.js.template

```diff
@@ -96,8 +96,17 @@ import { AuthenticationError, ForbiddenError, parseJWT } from '@redwoodjs/api'
  *     roles: parseJWT({ decoded: decoded }).roles,
  *   }
  * }
+ *
+ * @example - Cookies and other request information are available in the options paramter just in case
```

Comment:
`options` -> `req`

Reply:
Actually ;) `Cookies and other request information` seems to support calling it `req` in fact.